src/main/java/com/snowflake/kafka/connector/internal/SnowflakeConnectionServiceV1.java

```diff
@@ -1005,4 +1007,67 @@ public Connection getConnection() {
   public SnowflakeInternalStage getInternalStage() {
     return this.internalStage;
   }
+
+  @Override
+  public boolean updateStreamingChannelOffsetToken(
+      String tableName, String channelName, String offsetToken)
+      throws SnowflakeKafkaConnectorException {
+    InternalUtils.assertNotEmpty("tableName", tableName);
+    InternalUtils.assertNotEmpty("channelName", channelName);
+    String fullyQualifiedTableName =
```

Comment:
do we handle case sensitive names here?

Reply:
Good question, for instance I tried both 
select system$snowpipe_streaming_update_channel_offset_token('TESTDB_KAFKA.KAFKA_TEST.kafka_connector_test_table_3347672768933248582','test_connector_kafka_connector_test_table_3347672768933248582_2','');

and 
select system$snowpipe_streaming_update_channel_offset_token('TESTDB_KAFKA.KAFKA_TEST.KAFKA_CONNECTOR_TEST_TABLE_3347672768933248582','TEST_CONNECTOR_KAFKA_CONNECTOR_TEST_TABLE_3347672768933248582_2','');

It works for both. so case insensitive i guess.